### DISCUSSION TITLE
multiple asynchronous nvenc encoders

### DISCUSSION THREAD

**Discussion by sebionofrei** [obsproject/obs-studio#6127] [Development Questions] [answered]
Hello guys,

Using a plugin such as obs-multi-rtmp ... it seems that with 27.2.3 OBS can run only 2 asynchronous nvenc encoders. Trying to create a 3rd encoder also with nvenc would fail.
Any idea why? Is this a by-design functionality starting with 27.2.3 or am I missing something?

Many thanks.

**Comment by WizardCM** [comment, marked as answer]
This depends on your device, the driver you're running, and whether any other apps have their own NVENC sessions running.

https://www.techpowerup.com/268495/nvidia-silently-increases-geforce-nvenc-concurrent-sessions-limit-to-3

**Reply by sebionofrei** [reply]
Thanks for your answer.

However i find it intriguing that with 27.1.3 i was able to stream on 3 separate destinations having 3 separate re-encoding using nvenc ..
And now with 27.2.3, starting the 3rd one fails.

(can't recall if I ever tried more than 3 nvenc at the same time)

And this, without interfering with the nvidia drivers. Exactly on the day when upgraded OBS to 27.2.3 (due to the nagging twitch notification)... it starting behaving like this.
